server/pom.xml

```diff
@@ -277,16 +277,16 @@
       <artifactId>joda-time</artifactId>
       <version>${versions.jodatime}</version>
     </dependency>
-    <dependency>
-      <groupId>com.amazonaws</groupId>
-      <artifactId>aws-java-sdk-s3</artifactId>
-      <version>${versions.aws}</version>
-    </dependency>
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-math3</artifactId>
       <version>${versions.commonsmath}</version>
     </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>${versions.commonscodec}</version>
+    </dependency>
     <dependency>
```

Comment:
Skimmed through changes in server and didn't see a relevant change.


I guess some existing code requires it and it used to be provided transitively via aws dependency?

Reply:
> I guess some existing code requires it and it used to be provided transitively via aws dependency?

Yes, we use `Base32` from it in `PartitionName`